### DISCUSSION TITLE
[quadlet] [build] [auto-update]

### DISCUSSION THREAD

**Discussion by spmfox** [containers/podman#23812] [Q&A]
Hello - I've been embracing quadlet and converting all of my pods/containers into this format. I've been looking at the quadlet man pages for a few hours and I cannot figure out how `auto-update` works while using `.container` + `.build` quadlet units.

On the first run, it works perfectly - the image is built and the container starts. However, using the `AutoUpdate=local` option to the container does not actually re-invoke the `.build` service to re-build the container.

I'm  not sure if I'm missing something obvious, if this is still a work in progress, or if there is another way to do this (with quadlets). I assume I could create my own `.timer` to do this, but I'm trying to stay using pure quadlet only if possible.

Thank you.

**Comment by st0nie** [comment]
`AutoUpdate=local` means every time you re-build the image, your container will be re-created.
So if you restart `.build` unit, the `.container` will be restart.
It seems that `podman auto-update` cannot auto-rebuild your image. but you can set `Pull=always` in `[Build]` section, this allow you to auto-pull latest image while staring the `.build` unit
